packages/usa-tooltip/src/styles/_usa-tooltip.scss

```diff
@@ -1,5 +1,36 @@
 @use "uswds-core" as *;
 
+/// Create a spacer to increase target area for tooltip triangle.
+///
+/// @param {String} $direction - The direction of the tooltip; can be top, bottom, left, right.
+/// @param {Length} $triangle-size [5px] - The size of the triangle.
```

Comment:
Is it correct to define `$triangle-size` as a parameter here even though it's not being passed into the mixin?

Reply:
That's a good point. I don't think it's necessary since triangle size is already customizable.

I've moved the variable above the mixin in 04dcd6be4f to make it clearer that this mixin relies on that.